great_expectations/expectations/expectation.py

```diff
@@ -349,9 +355,17 @@ class Expectation(metaclass=MetaExpectation):
     expectation_type: ClassVar[str]
     examples: ClassVar[List[dict]] = []
 
-    def __init__(self, configuration: ExpectationConfiguration) -> None:
-        self._configuration = configuration
+    def __init__(self, **kwargs) -> None:
+        super().__init__(**kwargs)
+
+        # Everything below is purely to maintain current validation logic
+        expectation_type = camel_to_snake(self.__class__.__name__)
+        configuration = ExpectationConfiguration(
+            expectation_type=expectation_type,
+            kwargs=kwargs,
+        )
         self.validate_configuration(configuration)
+        self._configuration = configuration
```

Comment:
This validation logic will be moved over to Pydantic validators but needs to sit here for now. Additionally, I'm not sure we can easily remove `self._configuration`; I'm going to keep it around throughout this work and then circle back to remove it once we've met our primary objectives.